include/caf/sensor_manager.h

```diff
@@ -14,14 +14,19 @@ extern "C" {
 #include <zephyr/device.h>
 #include <zephyr/drivers/sensor.h>
 
+#define FLOAT_TO_SENSOR_VALUE(float_val)						\
+	{										\
+		.val1 = (int32_t)float_val,						\
+		.val2 = (int32_t)((float_val - (int32_t)float_val) * 1000000.0),	\
```

Comment:
Can we use the full range of the val2? Why stick to decimal numbers?

Reply:
Will use 1000000 due to implementation of sensor_value in zephyr, discussed offline.